dspace-spring-rest/src/main/resources/spring/spring-dspace-addon-validation-services.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<!-- The contents of this file are subject to the license and copyright detailed 
+	in the LICENSE and NOTICE files at the root of the source tree and available 
+	online at http://www.dspace.org/license/ -->
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd">
+
+	<bean name="metadataValidation" class="org.dspace.app.rest.submit.step.validation.MetadataValidation" scope="prototype">
```

Comment:
Can we move these beans (or the entire XML) to [dspace/config/spring/rest](https://github.com/DSpace/DSpace/tree/master/dspace/config/spring/rest)? Then they end up in the config folder of the dspace install directory which makes them easier to modify afterwards.

Reply:
So, I suggest to postpone it. In the next round I try to move the xml because it depends by the classpath for launch the batch script in DSpace and dependency for the other webapps to the new REST API (the dspace-rest jar need to compare in the lib of dspace installation and in the other webapp classpath (oai, sword, etc)